x-pack/plugins/apm/public/context/UrlParamsContext/index.tsx

```diff
@@ -52,7 +54,8 @@ export function urlParamsReducer(
 
 const UrlParamsContext = createContext({
   urlParams: {} as IUrlParams,
-  refreshTimeRange: defaultRefresh
+  refreshTimeRange: defaultRefresh,
+  uiFilters: {} as UIFilters
```

Comment:
Either `uiFilters` should be part of `urlParams`, or it should not be exposed by this context (and instead be in a separate `useUiFilters` hook).

Right now it's effectively `UrlParamsAndUiFiltersContext`...

Reply:
I'm warming up to the `useUiFilters` hooks, since it would also be able to return the stringified UiFilter like mentioned in https://github.com/elastic/kibana/pull/35778/files#r280032012

Reply:
that is a good point, i agree it makes sense in it's own hook